vpr/src/route/route_budgets.cpp

```diff
@@ -66,11 +66,11 @@ void route_budgets::free_budgets() {
     /*Free associated budget memory if set
      * if not set, only free the chunk memory that wasn't used*/
     if (set) {
-        free_net_delay(delay_min_budget, &min_budget_delay_ch);
-        free_net_delay(delay_max_budget, &max_budget_delay_ch);
-        free_net_delay(delay_target, &target_budget_delay_ch);
-        free_net_delay(delay_lower_bound, &lower_bound_delay_ch);
-        free_net_delay(delay_upper_bound, &upper_bound_delay_ch);
+        delay_min_budget.clear();
```

Comment:
clear does not free the memory, only sets the size to 0.  Is this the intention?.  shrink_to_fit increases the chance that the memory is free'd, and the capacity is set to 0.

Reply:
Good point, I've added vtr::release_memory(), which will force container memory to be released.